package.json

```diff
@@ -76,7 +76,7 @@
     "@veupathdb/prettier-config": "^1.0.0",
     "@veupathdb/react-scripts": "^1.0.1",
     "@veupathdb/tsconfig": "^1.0.1",
-    "@veupathdb/wdk-client": "^0.5.4",
+    "@veupathdb/wdk-client": "^0.5.5",
```

Comment:
You'll also want to change the `@veupathdb/wdk-client` `peerDependency` version from `>=0.5.4` to `>=0.5.5`. 

This will signal to `https://github.com/VEuPathDB/ClinEpiWebsite` (the repo where all the ClinEpi frontend assets are bundled together) that the bundled version of `@veupathdb/wdk-client`, declared [here](https://github.com/VEuPathDB/ClinEpiWebsite/blob/fea955c5332fce5ecb694d1ec67e3af952608753/Site/package.json#L14), must be at least `0.5.5`.

(In our various packages that depend on `@veupathdb/wdk-client`, we express the dependency of `@veupathdb/wdk-client` as a `peerDependency`, and not a regular `dependency`, to avoid introducing multiple copies of `@veupathdb/wdk-client`. More info about `peerDependencies` [here](https://flaviocopes.com/npm-peer-dependencies/).)